README.md

```diff
@@ -1,8 +1,43 @@
+
+## Skills
+
+**Proficient:** Java, RDBMS/SQL, Agile/Scrum, Linux/Bash, Git, C/C++
+
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database. The data was collected in real-time using a `crontab` job and persisted using PostgreSQL. SQL queries were run to perform analytics on the cluster such as creating groups of nodes and determining average memory consumption. The project was deployed using a docker container, and version control was maintained through GitHub.
+
+
+## Highlighted Projects
+**COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: Programmed a COVID-19 data tracker using C++ to allow for visual representations of worldwide COVID-19 trends. The project pulls COVID-19 data from API and the `iterator` object and `sort` function are used to parse and sort the data. The data is then graphed according to user input. This app issues graphical representation of trends such as countries with highest total case counts and countries with highest total death counts.
+
+**COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Programmed a software allowing businesses to track incidences of COVID-19 in their establishments and automate exposure notifications to their customer base. Implemented customer, business, and app admin interfaces to separate role-based privileges. Businesses were able to send out exposure notifications to customer groups, while customers were able to receive notifications and report COVID-19 cases. The data was persisted on an online database server and SQL queries were used to select customer groups affected by potential exposures. To notify customers of exposures emails were sent out via a pipe. A GUI was designed using the Qt toolkit. This project was a collaborative submission for an Object Oriented Programming course submission. The project lifecycle was managed using agile and scrum frameworks.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Worked on development and engineering projects in an agile enviornment. Worked on a scrum team, including taking on duties as a scrum master.
```

Comment:
Worked on development and engineering projects in an agile environment. Worked on a scrum team, including taking on duties as a scrum master.

Responsibilities
Agile
Soft skills

Reply:
Reworked this description to include more information